.babelrc

```diff
@@ -1,9 +1,8 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": [
-    "transform-object-rest-spread",
-    "transform-class-properties",
-    "transform-export-extensions",
-    "syntax-trailing-function-commas"
-  ]
+  "presets": [
+    ["es2015", { "loose": true }],
```

Comment:
It makes the transpiled code slightly less strictly spec-compliant, but faster/smaller:

Strict: http://babeljs.io/repl/#?evaluate=false&lineWrap=false&presets=es2015&experimental=true&loose=true&spec=false&code=import%20foo%20from%20'.%2Ffoo'%3B%0A%0Aclass%20A%20%7B%0A%20%20foo()%20%7B%7D%0A%7D%0A%0Aclass%20B%20extends%20A%20%7B%0A%20%20foo()%20%7B%7D%0A%20%20bar()%20%7B%7D%0A%7D%0A%0Aexport%20default%20B%3B%0A%0A%5B...a%2C%20b%5D%3B&playground=false
Loose: http://babeljs.io/repl/#?evaluate=false&lineWrap=false&presets=es2015-loose&experimental=true&loose=true&spec=false&code=import%20foo%20from%20'.%2Ffoo'%3B%0A%0Aclass%20A%20%7B%0A%20%20foo()%20%7B%7D%0A%7D%0A%0Aclass%20B%20extends%20A%20%7B%0A%20%20foo()%20%7B%7D%0A%20%20bar()%20%7B%7D%0A%7D%0A%0Aexport%20default%20B%3B%0A%0A%5B...a%2C%20b%5D%3B&playground=false

It's used pretty broadly – React and Relay both use it, as do most prominent community projects like Redux and React Router.